_elements/buttons.md

```diff
@@ -8,24 +8,24 @@ title: Buttons
   <!-- Add HTML markup for example here -->
 
   <h3>Primary Buttons</h3>
-  <button type="button">Button</button>
-  <button type="reset">Reset</button>
-  <button type="submit">Submit</button>
+  <button>Static</button>
+  <button class="usa-button-active">Active</button>
+  <button class="usa-button-hover">Hover</button>
 
   <h3>Secondary Buttons</h3>
-  <button class="usa-button-secondary" type="button">Button</button>
-  <button class="usa-button-secondary" type="reset">Reset</button>
-  <button class="usa-button-secondary" type="submit">Submit</button>
+  <button class="usa-button-secondary">Static</button>
+  <button class="usa-button-secondary usa-button-secondary-active">Active</button>
```

Comment:
We don't right now, but I was planning for the future, in case the primary active and secondary active states would look different. Like if someone chose different colors from what we have, so they may want more control.